File: Salario/Salario.java
import java.util.Scanner;

public class Salario{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int number = in.nextInt();
        int hours = in.nextInt();
        double money = in.nextDouble();
        System.out.printf("NUMBER = " + number + "\nSALARY = U$ %.2f\n", hours * money);
    }
}

File: Salario/Salario.cs
using System;

public class Salario{
    public static void Main(){
        int number = int.Parse(Console.ReadLine());
        int hours = int.Parse(Console.ReadLine());
        double money = double.Parse(Console.ReadLine());
        Console.WriteLine($"NUMBER = {number}\nSALARY = U$ {hours * money:0.00}");
    }
}